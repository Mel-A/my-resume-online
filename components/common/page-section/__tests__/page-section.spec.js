import { shallow } from 'enzyme'
import PageSection from '..'

describe('PageSection', () => {
  it('pageSection renders correctly', () => {
    const dummyClass = 'someClassName'
    const dummyText = 'someText'
    
    const pageSection = shallow(<PageSection className={dummyClass}>{dummyText}</PageSection>)
    expect(pageSection).toMatchSnapshot()
  })
})
