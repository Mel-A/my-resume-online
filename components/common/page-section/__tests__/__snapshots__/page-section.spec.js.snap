// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSection pageSection renders correctly 1`] = `ShallowWrapper {}`;
